doc/setup/install.rst

```diff
@@ -213,23 +272,42 @@ easily keeping (and running) multiple versions of Odoo side-by-side.
 There are two way to get the odoo source source zip or git.
 
 * Odoo zip can be downloaded from
-  https://nightly.odoo.com/9.0/nightly/src/odoo_9.0.latest.zip, the zip file
-  then needs to be uncompressed to use its content
+  https://nightly.odoo.com/9.0/nightly/src/odoo_9.0.latest.zip or the Download_
+  page, the zip file then needs to be uncompressed to use its content
 
-* git allows simpler update and easier switching between differents versions
+* git allows simpler update and easier switching between different versions
   of Odoo. It also simplifies maintaining non-module patches and
   contributions.  The primary drawback of git is that it is significantly
   larger than a tarball as it contains the entire history of the Odoo project.
 
-  The git repository is https://github.com/odoo/odoo.git.
-
+  The git repository is https://github.com/odoo/odoo.git for the Community
+  version. 
+  
   Downloading it requires a `git client <http://git-scm.com/download/>`_
   (which may be available via your distribution on linux) and can be performed
   using the following command:
 
   .. code-block:: console
 
       $ git clone https://github.com/odoo/odoo.git
+      
+  If you have access to the Enterprise repository, you can use this command 
+  to fetch the addons:
+
+  .. code-block:: console
+
+      $ git clone https://github.com/odoo/enterprise.git
+      
+  .. note:: This repository is only accessible to Partners and Enterprise 
+      customers. If you wish to get access, you can contact your sales representative
+      or our online support and provide us your Partnership/Enterprise code and
+      Github username.
+
+    
+.. warning:: Note that **the Enterprise repository doesn't contain the full Odoo source code** 
```

Comment:
I am wondering if we shouldn't put that kind of (important) information in an introduction section